.github/workflows/sync-opencraft-forks-with-upstream.yml

```diff
@@ -18,6 +18,10 @@ jobs:
             fork_branch: opencraft-release/palm.1
             upstream_repo: openedx/edx-platform
             upstream_branch: open-release/palm.master
+          - fork_repo: open-craft/edx-platform
+            fork_branch: opencraft-release/quince.1
+            upstream_repo: openedx/edx-platform
+            upstream_branch: open-release/quince.master
```

Comment:
Does it make sense to include other repos (like `frontend-app-learning`) here as well, @Agrendalath?

Reply:
@0x29a, sure, we can do this. Do you have a full list?

Reply:
@Agrendalath, I think we can add these as well:
```yml
- fork_repo: open-craft/frontend-app-learning
  fork_branch: opencraft-release/quince.1
  upstream_repo: https://github.com/openedx/frontend-app-learning
  upstream_branch: open-release/quince.master
- fork_repo: open-craft/frontend-app-gradebook
  fork_branch: opencraft-release/quince.1
  upstream_repo: https://github.com/openedx/frontend-app-gradebook
  upstream_branch: open-release/quince.master
- fork_repo: open-craft/frontend-app-authn
  fork_branch: opencraft-release/quince.1
  upstream_repo: https://github.com/openedx/frontend-app-authn
  upstream_branch: open-release/quince.master
```

There is also [frontend-app-ora-grading](https://github.com/open-craft/frontend-app-ora-grading/tree/opencraft-release/quince.1) Quince branch, but it's ahead of `open-release/quince.master`.

Reply:
@0x29a, thanks! I added these.